server/belga/search_providers.py

```diff
@@ -234,6 +234,10 @@ class Belga360ArchiveSearchProvider(superdesk.SearchProvider):
     def __init__(self, provider):
         super().__init__(provider)
         self.session = requests.Session()
+        self.content_types = {
+            c['_id']: c['_id']
```

Comment:
We do look up more often (`format_list_item` for each item) so `dict` is more effective here than `set`

Reply:
@idsvn why do you think `dict` is more effective? I think in our case it does not matter actually because we don't have millions of items there, so having `set` is logical.